test/testcppcheck.cpp

```diff
@@ -222,6 +223,15 @@ class TestCppcheck : public TestFixture {
         ASSERT_EQUALS(expected, cppcheck.getDumpFileContentsRawTokens(files, tokens1));
     }
 
+    void getDumpFileContentsLibrary() const {
+        ErrorLogger2 errorLogger;
+        CppCheck cppcheck(errorLogger, false, {});
+        cppcheck.settings().libraries.emplace_back("std.cfg");
+        std::vector<std::string> files{ "/some/path/test.cpp" };
+        const std::string expected = "  <library lib=\"std.cfg\"/>";
```

Comment:
The `  <library lib=\"` here is just redundant. You don't test that this is written in the dumpfile. We could change the code and write something totally different in the dumpfile and this test will still write this ..